### DISCUSSION TITLE
v4.0.9

### DISCUSSION THREAD

**Discussion by joschi** [dropwizard/dropwizard#9404] [Announcements]
## What's Changed
* Group Maven plugin updates in Renovate by @joschi in https://github.com/dropwizard/dropwizard/pull/9238
* chore(deps): update maven plugins (release/4.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9240
* fix(deps): update mockito monorepo to v5.13.0 (release/4.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9241
* chore(deps): update error_prone.version to v2.31.0 (release/4.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9245
* chore(deps): update github/codeql-action action to v3.26.6 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9249
* fix(deps): update dependency net.bytebuddy:byte-buddy to v1.15.1 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9250
* fix(deps): update dependency org.apache.commons:commons-lang3 to v3.17.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9255
* Use the SLF4J BOM to Manage SLF4J Versions by @JKomoroski in https://github.com/dropwizard/dropwizard/pull/9257
* chore(deps): update actions/upload-artifact action to v4.4.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9259
* fix(deps): update dependency org.eclipse.jetty:jetty-bom to v11.0.24 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9265
* chore(deps): update dependency sphinx-autobuild to v2024.9.3 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9266
* fix(deps): update dependency org.liquibase:liquibase-core to v4.29.2 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9270
* fix(deps): update dependency org.checkerframework:checker-qual to v3.47.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9271
* chore(deps): update dependency org.apache.maven.shared:maven-filtering to v3.4.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9279
* chore(deps): update dependency jp.skypencil.errorprone.slf4j:errorprone-slf4j to v0.1.28 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9283
* fix(deps): update dependency org.jboss.logging:jboss-logging to v3.6.1.final (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9284
* chore(deps): update actions/setup-java action to v4.3.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9285
* fix(deps): update dependency org.apache.tomcat:tomcat-jdbc to v10.1.29 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9290
* chore(deps): update dependency org.apache.maven.plugins:maven-gpg-plugin to v3.2.6 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9298
* chore(deps): update github/codeql-action action to v3.26.7 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9300
* fix(deps): update dependency org.apache.httpcomponents.core5:httpcore5 to v5.3 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9305
* fix(deps): update dependency net.java.dev.jna:jna to v5.15.0 (release/4.0.x) - autoclosed by @renovate in https://github.com/dropwizard/dropwizard/pull/9304
* chore(deps): update dependency com.uber.nullaway:nullaway to v0.11.3 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9310
* chore(deps): update dependency sphinx-autobuild to v2024.9.17 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9313
* fix(deps): update dependency org.apache.tomcat:tomcat-jdbc to v10.1.30 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9314
* chore(deps): update dependency sphinx-autobuild to v2024.9.19 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9321
* chore(deps): update github/codeql-action action to v3.26.8 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9325
* fix(deps): update guava monorepo to v33.3.1-jre (release/4.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9332
* chore(deps): update github/codeql-action action to v3.26.9 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9335
* chore(deps): update github actions (release/4.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9336
* chore(deps): update dependency org.cyclonedx:cyclonedx-maven-plugin to v2.8.2 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9351
* chore(deps): update dependency org.apache.maven.plugins:maven-gpg-plugin to v3.2.7 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9350
* Immediately release http client resources by @zUniQueX in https://github.com/dropwizard/dropwizard/pull/9296
* chore(deps): update junit5 monorepo to v5.11.1 (release/4.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9352
* fix(deps): update dependency net.bytebuddy:byte-buddy to v1.15.3 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9353
* fix(deps): update mockito monorepo to v5.14.0 (release/4.0.x) (minor) by @renovate in https://github.com/dropwizard/dropwizard/pull/9356
* chore(deps): update dependency org.apache.maven.plugins:maven-archetype-plugin to v3.3.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9354
* chore(deps): update dependency org.apache.maven.plugins:maven-javadoc-plugin to v3.10.1 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9371
* chore(deps): update github/codeql-action action to v3.26.10 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9372
* fix(deps): update mockito monorepo to v5.14.1 (release/4.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9374
* chore(deps): update dependency sphinx-autobuild to v2024.10.3 (release/4.0.x) - autoclosed by @renovate in https://github.com/dropwizard/dropwizard/pull/9375
* fix(deps): update dependency org.checkerframework:checker-qual to v3.48.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9377
* fix(deps): update dependency org.testcontainers:testcontainers-bom to v1.20.2 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9373
* fix(deps): update dependency io.dropwizard.metrics:metrics-bom to v4.2.28 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9387
* chore(deps): update github/codeql-action action to v3.26.11 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9390
* fix(deps): update dependency org.apache.httpcomponents.client5:httpclient5 to v5.4 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9329
* chore(deps): update actions/cache action to v4.1.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9395
* chore(deps): update junit5 monorepo to v5.11.2 (release/4.0.x) (patch) by @renovate in https://github.com/dropwizard/dropwizard/pull/9394
* fix(deps): update dependency com.fasterxml.jackson:jackson-bom to v2.18.0 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9355
* chore(deps): update dependency org.apache.maven.plugins:maven-surefire-plugin to v3.5.1 (release/4.0.x) by @renovate in https://github.com/dropwizard/dropwizard/pull/9399
* Use platform threads instead of virtual threads for thread pools by @zUniQueX in https://github.com/dropwizard/dropwizard/pull/9342

**Full Changelog**: https://github.com/dropwizard/dropwizard/compare/v4.0.8...v4.0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/dropwizard/dropwizard/releases/tag/v4.0.9'>v4.0.9</a>.</em>

**Comment by InnokentiyStruchkov** [comment]
A heads up for Kotlin users: https://github.com/FasterXML/jackson-module-kotlin/issues/832

When upgrading from Jackson 2.17.x to 2.18.0 check the `@JsonAnySetter` and `@field:JsonAnySetter` annotations are working as expected.
